### PR TITLE
RendezVous algorithm for hashing

### PR DESCRIPTION
**Pull request by ch40s3k**
Results of testing:
```
<ConsistentHashing with <function md5 at 0x10da26500> hash>
====
Distribution
18809 checks in alamo-scheduler1
19490 checks in alamo-scheduler2
span: 681
Took 192 miliseconds


<ConsistentHashing with <function sha256 at 0x10da265f0> hash>
====
Distribution
19158 checks in alamo-scheduler1
19141 checks in alamo-scheduler2
span: 17
Took 208 miliseconds


<ConsistentHashing with <function sha512 at 0x10da266e0> hash>
====
Distribution
20337 checks in alamo-scheduler1
17962 checks in alamo-scheduler2
span: 2375
Took 245 miliseconds


<RendezVous with <function md5 at 0x10da26500> hash>
====
Distribution
19235 checks in alamo-scheduler1
19064 checks in alamo-scheduler2
span: 171
Took 314 miliseconds


<RendezVous with <function sha256 at 0x10da265f0> hash>
====
Distribution
19217 checks in alamo-scheduler1
19082 checks in alamo-scheduler2
span: 135
Took 324 miliseconds


<RendezVous with <function sha512 at 0x10da266e0> hash>
====
Distribution
19059 checks in alamo-scheduler1
19240 checks in alamo-scheduler2
span: 181
Took 404 miliseconds
```

Winner: RendezVous with md5 hashing (time to span)

### CODE REVIEW

File: alamo_scheduler/hashing.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import hashlib
+
+
+def md5(key):
+    return int(hashlib.md5(key).hexdigest(), 16)
+
+
+class RendezVous(object):
```
Comment (deejay1) [inline alamo_scheduler/hashing.py]: maybe some doc strings? like how this should work, what's based on?

Comment (ch40s3k) [inline alamo_scheduler/hashing.py]: https://en.wikipedia.org/wiki/Rendezvous_hashing

Comment (ch40s3k) [inline alamo_scheduler/hashing.py]: Added info in README.md

File: alamo_scheduler/hashing.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import hashlib
+
+
+class Hashing(object):
+
+    def __init__(self, nodes=None):
+        if nodes is None:
+            nodes = []
+        self.nodes = nodes
```
Comment (pnuckowski) [inline alamo_scheduler/hashing.py]: `self.nodes = nodes or []`
or 
`self.schedulers = schedulers or []`

Comment (ch40s3k) [inline alamo_scheduler/hashing.py]: OK

**Pull request merged**